tests/account.write.test.ts

```diff
@@ -20,9 +24,15 @@ import {
   topUp
 } from "./src/testUtils"
 import type { MasterClient, NetworkConfig } from "./src/testUtils"
+import { GENERIC_FALLBACK_SELECTOR_SELECTOR } from "../src/bundler/utils/Constants"
 
 const NETWORK_TYPE: TestFileNetworkType = "FILE_LOCALHOST"
 
+// todo
+// remove hard code and fetch from deployments
```

Comment:
@joepegler was asking for this part. probably need help here and update after our all deployments are sourced in a chosen way.